frontend/src/metabase/api/collection.ts

```diff
@@ -21,7 +29,32 @@ export const collectionApi = Api.injectEndpoints({
       providesTags: (response, error, { models }) =>
         provideCollectionItemListTags(response?.data ?? [], models),
     }),
+    getCollection: builder.query<Collection, CollectionRequest>({
+      query: ({ id, ...body }) => ({
+        method: "GET",
+        url: `/api/collection/${id}`,
+        body,
+      }),
+      providesTags: collection =>
+        collection ? provideCollectionTags(collection) : [],
```

Comment:
wonder if `provideCollectionTags` and `provideCollectionListTags` could have the conditional baked in so that future folks don't have to remember to do it?

Reply:
I don't disagree, but I'm following the pattern of every single other tag provider 😬

Reply:
🤡 i probably started that... fair enough